signalbot/bot.py

```diff
@@ -73,11 +73,16 @@ def __init__(self, config: dict):
             self._redis_port = config_storage["redis_port"]
             self.storage = RedisStorage(self._redis_host, self._redis_port)
         except Exception:
-            self.storage = InMemoryStorage()
-            logging.warning(
-                "[Bot] Could not initialize Redis. In-memory storage will be used. "
-                "Restarting will delete the storage!"
-            )
+            try:
+                self._sqlite_db = config_storage["sqlite_db"]
```

Comment:
Not sure about this. If we are adding a new database type, would it make more sense to add a new entry to the config? Something like:

```python
config_redis = {storage: {type: "redis", redis_host: "redis", redis_port: 123}}
redis_bot = SignalBot(config=config_redis)

config_sqlite = {storage: {type: "sqlitle", sqlite_host: "sqlite"}}
sqlite_bot = SignalBot(config=config_sqlite)
```
where this code then becomes
```python
if "type" not in config["storage"] or config["storage"]["type] == "redis:
  # do redis as before
elif config["storage"]["type] == "sqlite":
  # do sqlite
else:
  # do in-memory sqlite
```

Reply:
I added the check for an sqlite "type" field as suggested, and left the rest of the code as it was before my change.